go.mod

```diff
@@ -42,15 +42,15 @@ require (
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect; indirects
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.1
-	github.com/stretchr/testify v1.6.1
+	github.com/stretchr/testify v1.7.0
```

Comment:
not sure why, but `go mod tidy` updates the version for me

Reply:
I'll redo the PR as last bit of the release